<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelWidget</class>
 <widget class="QWidget" name="ChannelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
background-color: transparent;
color: #FFFFFF;
padding: 0px;
min-height: 0px;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_channel">
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyLabel" name="label_name">
      <property name="text">
       <string>CAM1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="toolButton_clear">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
border: 0px;
border-radius: 3px;
padding: 3px;
}
QToolButton:hover
{
background-color: rgba(255, 255, 255, 100);
}
QToolButton:pressed
{
background-color: rgba(255, 255, 255, 50);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/layout/layout/close.png</normaloff>:/layout/layout/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_index">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
